### PR TITLE
Updates the language in the aside in the responiveness metric doc.

### PR DESCRIPTION
**Pull request by malchata**
Changes proposed in this pull request:

- Updates the language in the aside at the top of the [responsiveness metric feedback post](https://web.dev/responsiveness/) with more specific information.

cc: @anniesullie @powdercloud for thoughts/guidance on language.

### CODE REVIEW

File: src/site/content/en/blog/responsiveness/index.md [hunk 1]
```diff
@@ -14,9 +14,7 @@ tags:
 ---
 
 {% Aside 'important' %}
-A candidate for a new responsiveness metric has been chosen, so feedback is no
-longer necessary at this time. When finalized, this metric will be documented
-with its own landing page. This space will be updated when that happens.
+We have chosen option 2 of [high quantile approximation](/responsiveness/#high-quantile-approximation) as a candidate for the new responsiveness metric, with [maximum event duration](/responsiveness/#measure-interaction-latency) being the definition for event interaction. The `experimental.responsiveness` table in the February 2022 CrUX dataset will include this new metric when it releases on 8 March, 2022.
```
Comment (philipwalton) [inline src/site/content/en/blog/responsiveness/index.md:17]: A few comments on this line:
- s/a candidate/the candidate/
- s/being the definition for event interaction/as the definition for interaction latency/

Also, nit: I don't think `experimental.responsiveness` is a table, so you might want to rephrase that as something like: "This metric will be available as `experimental.responsiveness` starting  in the February 2022 [CrUX BigQuery](https://developers.google.com/web/tools/chrome-user-experience-report/bigquery/getting-started) dataset, released on March 8, 2022.

**Review by philipwalton — APPROVED** [review]
LGTM, thanks!

**Comment by anniesullie** [comment]
LGTM!

**Review by Demirsa — COMMENTED** [review]
[CODE_OF_CONDUCT.md](https://github.com/GoogleChrome/web.dev/files/9107920/CODE_OF_CONDUCT.md)

**Pull request merged**